src/subgraph/orders/index.ts

```diff
@@ -25,8 +25,21 @@ export const getAllOrdersByUserAddress = async (
 ): Promise<Order[]> => {
   try {
     const subgraphResponse: any = await request(subgraphEndpoint, fetchOrdersByUserQuery(userAddress, count, skip));
+    if (!subgraphResponse) throw new Error('Error While Fechting All Order By user Address');
+    const accountIdArray = subgraphResponse?.orders?.map((order: Order) => {
+      return order?.snxAccount?.id;
```

Comment:
Incorrect logic. Have we tested this? @Akhilleshgoswami

Reply:
Please run the test cases and verify valid data response

Reply:
yes